quickstarts/WFLY-16140_Remove_outdated_Quickstarts.adoc

```diff
@@ -0,0 +1,240 @@
+= WFLY-16140 - Remove Outdated Quickstarts
+:author:            Eduardo Martins
+:email:             emartins@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The current set of Quickstarts is considered excessive (102), with multiple variants of the same Quickstart (e.g. 5 kitchensink, 11 helloworld); and very outdated, with many quickstarts covering old and/or no longer functionalities.
+
+Also, moving forward we are turning the focus to the Developer Guide, going for a major rework of it, and the main role of the Quickstarts is to simply provide the ready-to-try apps, which are referenced by such a document. The new guide structure is simple, the user starts by developing an hello world from scratch on bare metal, and same for OpenShift, what could be seen as the first part of the guide, which is for first timer/beginner developers. This first part also has sections to take the hello world to specific technologies, such as EJB, messaging, or web services. The second part of the guide is for advanced users, covering advanced topics for multiple technologies supported by EAP, where we think most of the existent content from the EAP Developer Guide and QuickStarts guides will go to. Taking into account this kind of guide structure, the quickstarts to keep/upgrade should mainly be the ones that are relevant/needed for the document.
```

Comment:
Is the guide going to live in the quickstart repository?

Maybe a bit off-topic for this proposal, but we could have a nice or even the a build of the current quickstart docs hosted on the quickstart project. It would then be available via https://docs.wildfly.org/quickstart

Reply:
No, I think we should keep the developer guide where it is, it's just a major content rework.

Reply:
I assume by this you mean the https://docs.wildfly.org/26/Developer_Guide.html, correct?